common/djangoapps/course_modes/models.py

```diff
@@ -692,3 +697,38 @@ class CourseModesArchive(models.Model):
     expiration_date = models.DateField(default=None, null=True, blank=True)
 
     expiration_datetime = models.DateTimeField(default=None, null=True, blank=True)
+
+
+class CourseModeExpirationConfig(ConfigurationModel):
+    """
+    Configuration for time period from end of course to auto-expire a course mode.
+    """
+
+    # TODO: Django 1.8 introduces a DurationField
```

Comment:
Since you're on the Django 1.8 branch, this should be replaced with a `DurationField`.